### PR TITLE
Update codegen docs to include Javascript

### PR DESCRIPTION
**Pull request by anthonylusardi-da**
In progress, ~~pending some clarification here: https://discuss.daml.com/t/can-daml-codegen-js-options-be-specified-in-daml-yaml/648~~

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Comment by anthonylusardi-da** [comment]
This is generally good to go, but question, is `1.0.1` still an appropriate version to have for the Java/Scala dependencies? Might as well fix that here if not.

As in:
```
For **Scala**, you can depend on::

    "com.daml" %% "bindings-scala" % 1.0.1

For **Java**, add the following **Maven** dependency::

    <dependency>
      <groupId>com.daml</groupId>
      <artifactId>bindings-java</artifactId>
      <version>1.0.1</version>
    </dependency>
```

File: docs/source/tools/codegen.rst [hunk 1]
```diff
@@ -7,37 +7,31 @@ DAML codegen
...
-Help for **DAML to Scala** codegen::
-
-  $ daml codegen scala --help
-
-Both **DAML to Java** and **DAML to Scala** take the same set of configuration settings::
-
-    Usage: codegen [options] <DAR-file[=package-prefix]>...
-
-    Code generator for the DAML ledger bindings.
+**Java** and **Scala** take the same set of configuration settings::
```
Comment (hurryabit) [inline docs/source/tools/codegen.rst:34]: I'm not sure it is clear that we're referring to the codegens to Java/Scala here.

File: docs/source/tools/codegen.rst [hunk 2]
```diff
@@ -7,37 +7,31 @@ DAML codegen
 Introduction
 ============
 
-You can use the DAML codegen to generate Java and Scala classes representing DAML contract templates. These classes incorporate all boilerplate code for constructing corresponding ledger :ref:`com.daml.ledger.api.v1.CreateCommand` and :ref:`com.daml.ledger.api.v1.ExerciseCommand`.
+You can use the DAML codegen to generate Java and Scala classes representing DAML contract templates.
```
Comment (cocreature) [inline docs/source/tools/codegen.rst:10]: You start by only talking about Java and Scala but then when it comes to running the codegen you introduce JavaScript.

File: docs/source/tools/codegen.rst [hunk 3]
```diff
@@ -7,37 +7,31 @@ DAML codegen
...
+These classes incorporate all boilerplate code for constructing corresponding ledger ``com.daml.ledger.api.v1.CreateCommand``
```
Comment (cocreature) [inline docs/source/tools/codegen.rst:11]: Also `ExerciseByKey` and `CreateAndExercise`.

**Review by cocreature — COMMENTED** [review]
LGTM with a minor comment on the intro. Do you still want to merge this @anthonylusardi-da?

**Comment by anthonylusardi-da** [comment]
One last thing, is `1.0.1` still the right version for Java/Scala deps?

> This is generally good to go, but question, is `1.0.1` still an appropriate version to have for the Java/Scala dependencies? Might as well fix that here if not.

File: docs/source/tools/codegen.rst [hunk 4]
```diff
@@ -79,29 +84,22 @@ The above settings can be configured in the ``codegen`` element of the DAML proj
...
-    $ daml codegen scala ./.daml/dist/quickstart-0.0.1.dar=com.daml.quickstart.iou --output-directory=scala-codegen/src/main/scala --verbosity=2
+In order to compile the resulting **Java** or **Scala** classes, you need to
+add the corresponding dependencies to your build tools. 
 
-In order to compile the resulting **Scala** or **Java** classes, you need to
-add the corresponding dependencies to your build tools. For **Scala**, you can
-depend on::
+For **Scala**, you can depend on::
 
     "com.daml" %% "bindings-scala" % 1.0.1
```
Comment (cocreature) [inline docs/source/tools/codegen.rst:100]: Either update it to 1.5.0 and add a comment that this should be updated to whatever SDK version you are using or use a placeholder and tell users to replace it.

**Review by cocreature — APPROVED** [review]
Thanks!

**Pull request merged**